frontend/README.md

```diff
@@ -140,3 +138,50 @@ swagger-codegen-cli@2.4.7, which you can get
 [here](https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.7/).
 Make sure the jar file is somewhere on your path with the name
 swagger-codegen-cli.jar, then run `npm run apis`.
+
+## MLMD components
+
+* `src/mlmd` - components for visualizing data from an `ml-metadata` store. For more information see the
+ [kubeflow/metadata](https://github.com/kubeflow/metadata) repository.
```

Comment:
nit: update link to github.com/google/ml-metadata, because kubeflow/metadata is no longer maintained